cspell/.cspell.json

```
@@ -864,11 +871,13 @@
         "toplevel",
         "Toprated",
         "tostring",
+        "tqdm",
         "trafficlight",
         "trafficlights",
         "traj",
         "trajs",
         "turtlebot",
+        "twinx",
```

```suggestion
        "twinx",
        "twiny",
```
Add a related word. <https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.twiny.html?highlight=twin#matplotlib.axes.Axes.twiny>